Lib/multiprocessing/shared_memory.py

```diff
@@ -305,15 +305,20 @@ def _extract_recreation_code(value):
         else:
             return 3  # NoneType
 
+    def _get_str_or_bytes_format(self, item):
+        """Used to ensure the correct length is inserted into the
+        the formatted _types_mapping for multibyte utf-8 characters."""
+        length = len(item if isinstance(item, bytes) else item.encode())
```

Comment:
Shouldn't this explicitly encode to `utf-8` rather than relying on the default?

Reply:
which appears to be referenced as the global `_encoding` below.

Reply:
Good catch, thank you!